src/partials/connectors-menu.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="js-connector-menu" style="display: none;">
```

Comment:
The connectors-menu div will need to be inside the row or it will need to be populated from information in the row when it appears. Right now, it's static, so it's the same content for every row.